resource-watch/doc-api
# Improvements to query docs

## Summary

This PR refers to [GTC-506](https://gfw.atlassian.net/browse/GTC-506). It adds:
* SQL compatibility tables for the dataset providers ArcGIS and GEE
* Description of PostGIS functions for CartoDB datasets
* Usage of `geostore` as query parameter when executing queries.
* `DELETE` queries support for document-based datasets.

## Testing instructions

If you feel like deep-diving into the contents of the PR, follow the steps below:

1. Clone the documentation repository and follow the [installation instructions on the README](https://github.com/resource-watch/doc-api#installation-and-usage).
2. Checkout the branch of this PR: `docs/query-missing-providers`
3. Check out the SQL compatibility tables for ArcGIS and GEE.
